### ISSUE TITLE
Count challenges you've created in your total

### ISSUE THREAD

**Issue by degoeym** [paircolumbus/progress#20]
Feels a bit odd that it says I haven't completed the BootstrapChallenge item when I was the one who creaetd it. :stuck_out_tongue_closed_eyes:

**Comment by nicolasmccurdy** [comment]
I agree, this would be nice! I thought about this a while ago but I kept forgetting.
